### PR TITLE
feat: remove bourbon

### PR DESCRIPTION
**Pull request by frassinier**
Add Autoprefixer instead of Bourbon to be complient with Webpack 2 and the future!

### CODE REVIEW

File: src/components/dropdown-menu/_dropdown-menu.scss [hunk 1]
```diff
@@ -38,11 +38,20 @@ sc-dropdown-menu {
...
+      text-overflow: ellipsis;
+      white-space: nowrap;
+      word-wrap: normal;
     }
 
     &:after {
-      @include position(absolute, 0 5px auto auto);
+      position: absolute;
+      top: 0;
+      left: 5px;
```
Comment (charlesnguyen) [inline src/components/dropdown-menu/_dropdown-menu.scss]: right instead of left.

File: src/components/dropdown/_dropdown.scss [hunk 2]
```diff
@@ -30,7 +30,11 @@ sc-dropdown {
     position: relative;
 
     &:after {
-      @include position(absolute, 0 5px auto auto);
+      position: absolute;
+      top: 0;
+      left: 5px;
```
Comment (charlesnguyen) [inline src/components/dropdown/_dropdown.scss]: right instead of left.

File: package.json [hunk 3]
```diff
@@ -74,6 +68,7 @@
     "node-sass": "^3.4.2",
     "phantomjs-polyfill": "0.0.1",
     "phantomjs-prebuilt": "^2.1.4",
+    "postcss-cli": "^3.0.0-beta",
```
Comment (charlesnguyen) [inline package.json:71]: It is used to execute autoprefixer?

File: package.json [hunk 4]
```diff
@@ -5,25 +5,19 @@
...
-      "scripts:dev": "webpack --config webpack.config.dev.js",
-      "scripts:prod": "webpack --config webpack.config.prod.js",
-
+    "scripts:dev": "webpack --config webpack.config.dev.js",
+    "scripts:prod": "webpack --config webpack.config.prod.js",
     "styles": "npm run styles:dev && npm run styles:prod",
-      "styles:dev": "node-sass --include-path ./node_modules/bourbon/app/assets/stylesheets/ src/sunchoke.full.scss dist/sunchoke.css",
-      "styles:prod": "node-sass --include-path ./node_modules/bourbon/app/assets/stylesheets/ --output-style compressed src/sunchoke.full.scss dist/sunchoke.min.css",
-
+    "styles:dev": "node-sass src/sunchoke.full.scss dist/sunchoke.css && ./node_modules/postcss-cli/bin/postcss -u autoprefixer -b 'last 2 versions' -r dist/*.css",
```
Comment (charlesnguyen) [inline package.json:14]: I can not run this. maybe windows problem?
Error:
Rendering Complete, saving .css file...
Wrote CSS to C:\Users\cnguyen\DataPrep\sunchoke\dist\sunchoke.css
'.' n'est pas reconnu en tant que commande interne
ou externe, un programme exécutable ou un fichier de commandes.

**Review by charlesnguyen — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (frassinier) [reply package.json:71]: Yep

File: src/components/dropdown-menu/_dropdown-menu.scss (hunk 1 above)
Comment (frassinier) [reply src/components/dropdown-menu/_dropdown-menu.scss]: Good catch! :+1:

File: src/components/dropdown/_dropdown.scss (hunk 2 above)
Comment (frassinier) [reply src/components/dropdown/_dropdown.scss]: Good catch! :+1:

File: package.json (hunk 4 above)
Comment (frassinier) [reply package.json:14]: Yes, it's designed to be executed with unix operating system :/

Could you try `node-sass src/sunchoke.full.scss dist/sunchoke.css && postcss -u autoprefixer -b 'last 2 versions' -r dist/*.css`?

**Review by charlesnguyen — APPROVED** [review]

**Pull request merged**